static/app/components/sidebar/sidebarAccordion.tsx

```diff
@@ -14,15 +27,44 @@ type SidebarAccordionProps = SidebarItemProps & {
   children?: React.ReactNode;
 };
 
+export const ExpandedContext = createContext<{
+  openMainItemId: string | null;
+  setOpenMainItem: (mainItemId: string | null) => void;
```

Comment:
Definitely fine, _but_ would be better if the names matched the name of the provider. Otherwise, you're providing _expanded context_ but you're setting _main item ID_? A little strange

Reply:
👍🏻